spec/requests/api/v1/items_request_spec.rb

```diff
@@ -136,6 +136,34 @@
      
       expect(result).to eq(expected[:data])
     end
+
+    it 'can return a single merchant by an items ID' do
+      id = @item1.id
+      expected = {
+        data: {
+            id: @KozeyGroup.id.to_s,
+            type: "merchant",
+            attributes: {
+                name: @KozeyGroup.name
+            }
+        }
+    }
+      get "/api/v1/items/#{id}/merchant"
+      
+      expect(response).to be_successful
+
+      result = JSON.parse(response.body, symbolize_names: true)[:data]
+
+      expect(result).to eq(expected[:data])
+    end
+
+    it 'renders proper 404 response if item id does not exist when returning a single merchant' do
+      get "/api/v1/items/87453487579348534987789234789/merchant"
+
+      expect(response.status).to eq(404)
+      expect(response.body).to eq("{\"error\":\"404: Item Not Found\"}")
```

Comment:
TBD if we are still getting these errors, may need to be refactored, time will tell.